### ISSUE TITLE
ui idea

### ISSUE THREAD

**Issue by SucculentJohn** [clifford269/FrutigerFX#8]
I like this theme, it looks beautiful and its a fresh breath of air after having to use the nasty firefox metro default theme. 

What i think could be worked on is the ui itself. it does capture a few bits of the skeumorphism/aero interface greatly but it is also a mish-mash of other ui themes which in my opinion makes it a little odd to see at first.

It mixes the windows vista ui with bits and pieces of the mac ui and windows 7 ui and it looks a little weird with the metro icons.
![faggot](https://github.com/user-attachments/assets/5a5ed08d-c1f2-4cdf-b17d-6f2d9ad3f096)

This is not in any way shape or form written with the intent to throw all the hard work you've done into the gutter, it''s just a my personal opinione

TL;DR : I am very picky and annoying

**Comment by clifford269** [comment]
Yeah actually, I was running out of the fuel to work on FrutigerFX of late. There has been literally zero help from any other developer and I have to redirect my energy towards truly rewarding feats. Neither has this project been of any monetary benefit to me.

**Comment by duckduckgroose** [comment]
Hey, this is amazing!! Thank you so much for making this!!!
Just installed it yesterday. However, I'm having some issues getting the UI to display properly. 
Any advice? The biggest problems are icons overlapping on tabs and menus. Here's a screenshot:

![Image](https://github.com/user-attachments/assets/2abd008c-97df-425b-9512-f05e629e5133)
